Samples/Asp.net Core - Support Microsoft/Asp.net Core - Support Microsoft/README.md

```diff
@@ -0,0 +1,94 @@
+# GPT Pro: Plugin Microsoft Support
+
+### Microsoft Support
+
+This example contains a web API used to contact Microsoft Support (https://support.microsoft.com) and return the first response in JSON.
+
+#### How To Run This Sample in a local environment
+
+##### Required Tools
+
+- [Visual Studio](https://visualstudio.microsoft.com/fr/downloads/)
+- We recommand to use [ngrok](https://ngrok.com/)
```

Comment:
plutot que de mettre we recommand, tu peux mettre '[ngrok] or any other tunneling tool'